packages/cli/src/controllers/ldap.controller.ts

```diff
@@ -11,11 +12,15 @@ import { BadRequestError } from '@/errors/response-errors/bad-request.error';
 @Authorized()
 @RestController('/ldap')
 export class LdapController {
-	constructor(
-		private ldapService: LdapService,
-		private ldapSync: LdapSync,
-		private internalHooks: InternalHooks,
-	) {}
+	private ldapService: LdapService;
+
+	private ldapSync: LdapSync;
+
+	constructor(private readonly internalHooks: InternalHooks) {
+		const { service, sync } = LdapManager.getInstance();
+		this.ldapService = service;
+		this.ldapSync = sync;
+	}
```

Comment:
Why did we need to undo DI here?

Reply:
This did not have DI. `LdapSync` and `LdapService` aren't actually decorated, and can't be injected like that. 
I have a PR fixing that a few other things around how the LDAP code is setup.